src/store/cache/user/graph/formatter.ts

```diff
@@ -38,16 +38,15 @@ export const formatUser = async (
   if (user.services.length === 0 && user.stakeAmount === '0')
     return formattedUser
 
+  // Note: We must make a contract call to check "validBounds" because that value is not
+  // updated with the graph
+  const serviceProvider: ServiceProvider = await aud.ServiceProviderClient.getServiceProviderDetails(
```

Comment:
any noticeable impact on load?

Reply:
Don't think so, I didn't notice when I was playing around w/ it.